integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForDeleteCommand.scala

```diff
@@ -104,24 +111,27 @@ private[sql] case class CarbonProjectForDeleteCommand(
       // handle the clean up of IUD.
       CarbonUpdateUtil.cleanUpDeltaFiles(carbonTable, false)
 
-      val deletedSegments = DeleteExecution.deleteDeltaExecution(
+      val (deletedSegments, deletedRowCountTemp) = DeleteExecution.deleteDeltaExecution(
         databaseNameOp,
         tableName,
         sparkSession,
         dataRdd,
         timestamp,
         isUpdateOperation = false,
         executorErrors)
+
+      // check whether it fails before execute horizontal compaction
```

Comment:
This comment is not required here, if you want to add one , then it can be changed to `Check for any failures occured during delete delta execution`

Reply:
Done